src/components/progress-tracker/progress-tracker.stories.mdx

```diff
@@ -157,256 +168,54 @@ This is an example of a `ProgressTracker` component with `currentProgressLabel`
       style={{ display: "flex", flexDirection: "column", alignItems: "center" }}
     >
       <ProgressTracker
-        progress={25}
-        currentProgressLabel="$50"
-        maxProgressLabel="$200"
+        progress={50}
+        currentProgressLabel="$50 of $200"
       />
       <ProgressTracker
         mt={2}
-        progress={50}
-        currentProgressLabel="$100"
-        maxProgressLabel="$200"
+        progress={70}
+        showDefaultLabels
+        currentProgressLabel="Step 3 of 5"
+        description="Adding VAT"
       />
       <ProgressTracker
         mt={2}
         progress={100}
         currentProgressLabel="$200"
-        maxProgressLabel="$200"
+      />
+      <ProgressTracker
+        mt={2}
+        progress={100}
+        error={true}
+        currentProgressLabel="error"
       />
     </div>
   </Story>
 </Canvas>
 
-It also possible to render the component with a mix of default and custom labels by only setting one of the `currentProgressLabel`
+It is also possible to render the component with a mix of default and custom labels by only setting one of the `currentProgressLabel`
 and `maxProgressLabel` props and optionally setting the `showDefaultLabels` prop.
 
 <Canvas>
   <Story name="default and custom label values">
     <div
       style={{ display: "flex", flexDirection: "column", alignItems: "center" }}
     >
-      <ProgressTracker progress={15} currentProgressLabel="$50" />
-      <ProgressTracker mt={2} progress={50} maxProgressLabel="$200" />
+      <ProgressTracker progress={50} currentProgressLabel="50%" />
+      <ProgressTracker mt={2} progress={50} currentProgressLabel="Step 3 of 5" />
```

Comment:
While this visually achieves the original design, I don't think this is the right approach here, since `currentProgressLabel` was designed to represent the current level of progress, not both the current and completed levels - in other words "Step 3" should really be passed to `currentProgressLabel` and "5" to `maxProgressLabel`.

Personally I think we would need to update the existing labelling props and release this update as a breaking change, since the current props don't seem to work well with the redesign you have been given.

One potential solution would be to split `showDefaultLabels` into two separate props - one for the current level of progress, and the other for the completed level of progress. That way we would be able to handle the case where both labels need to be rendered with the "of" keyword between them.

@nicktitchmarsh @edleeks87 what do you think about this?

Reply:
I think that a potential solution looks like a great idea.

Reply:
If we were building from scratch James approach sounds reasonable, but as it's an existing component I'd like to avoid a breaking change. I don't really mind Debra's approach of using the `currentProgressLabel` but we could update it to be a `node` so that they compose it with `Typography` components to achieve the above

Reply:
Can you elaborate a bit more on your approach @edleeks87 ?